lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -254,6 +257,42 @@ export const FeatureInfoSection = observer(
         this.renderDataTitle();
       const reactInfo = getInfoAsReactComponent(this);
 
+      let threddsChart = null;
```

Comment:
This is the section of the PR that most concerns me. It generates a pixel drill chart in the feature info that is pretty specific to THREDDS.

Reply:
no longer relevant